pkgs/development/interpreters/python/mk-python-derivation.nix

```diff
@@ -85,6 +84,7 @@ toPythonModule (python.stdenv.mkDerivation (builtins.removeAttrs attrs [
   # Python packages don't have a checkPhase, only an installCheckPhase
   doCheck = false;
   doInstallCheck = doCheck;
+  installCheckInputs = checkInputs;
```

Comment:
Couldn't this be wrapped in a `lib.optionals`?

Reply:
It could, but it doesn't matter, the `mkDerivation` takes care of it.